javascript/chronometer.js

```diff
@@ -1,26 +1,28 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => this.currentTime++, 1000);
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60);
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(number) {
+    number = number.toString();
+    return number.length !== 2 ? "0" + number : number;
   }
```

Comment:
Nice, another way you can write it is by using template stings so you won't need line 16 
```
return number.length !== 2 ? `0${number}` : `${number}`;